airbyte-webapp/src/views/Connection/TransformationForm/TransformationForm.tsx

```diff
@@ -32,7 +32,10 @@ const validationSchema = yup.object({
   name: yup.string().required("form.empty.error"),
   operatorConfiguration: yup.object({
     dbt: yup.object({
-      gitRepoUrl: yup.string().required("form.empty.error"),
+      gitRepoUrl: yup
+        .string()
+        .required("form.empty.error")
+        .matches(/((http(s)?)|(git@[\w.]+))(:(\/\/)?)([\w.@:/\-~]+)(\.git)$/, "form.repositoryUrl.invalidUrl"),
```

Comment:
Regex looks good to me, but IMO would be a good place for a unit test. Since the backend apparently fails without this, it would be good to defend against a regression. Maybe `validationSchema` could be exported and tested explicitly with `validationSchema.validate( ... )`?

Reply:
Yeah, sounds reasonable. 
Will add tests for `validationSchema`